src/main/java/org/mockito/internal/util/junit/JUnitFailureHacker.java

```diff
@@ -6,26 +6,73 @@
 
 import org.junit.runner.notification.Failure;
 import org.mockito.internal.exceptions.ExceptionIncludingMockitoWarnings;
-import org.mockito.internal.util.reflection.Whitebox;
 
+import java.lang.reflect.Field;
+
+@Deprecated
```

Comment:
It's an internal class and is only used in the VerboseMockitoRunner. Therefore I do not think documentation is required.